clonelists/Sony - PlayStation 2.json

```diff
@@ -71,11 +71,13 @@
 	},
 
 	"removes": {
+		"ATV Offroad Fury 4 (USA) (Layer 0)": {},
```

Comment:
Usually I try to avoid removes unless I have no other choice. In this case, I'd add this as a clone to ATV Offroad Fury 4, with a priority of 2 (although this is overkill, as Retool should prioritize it lower anyway as a beta).

Reply:
Updated.